doc/content/integrations/mqtt-clients/hivemq/_index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "HiveMQ"
+description: ""
+weight: 
+---
+
+[HiveMQ](https://www.hivemq.com/) is an MQTT broker and a client based messaging platform which uses MQTT protocol for fast, reliable and efficient bi-directional data transfer to and from IoT devices. HiveMQ provides its own client library, but it can be used with any MQTT compliant client library. It can be deployed on a private, hybrid or public cloud. You can integrate HiveMQ with existing enterprise systems thanks to its open API and a flexible extension framework.
+
+<!--more-->
+
+HiveMQ also offers an open source tool called [MQTT CLI](https://github.com/hivemq/mqtt-cli), which provides a command line interface to interact with MQTT brokers. This tool can be used in a shell mode, allowing you to use multiple MQTT clients simultaneously. 
+
+>Note: HiveMQ broker is compliant with the MQTT 3.1, 3.1.1 and 5.0 protocol specifications, while the MQTT CLI tool supports 3.1.1 and 5.0 versions.
+
+This guide contains the instructions to use HiveMQ CLI tool in a shell mode for subscribing and publishing to topics used by {{% tts %}} [MQTT Server]({{< ref "/integrations/mqtt" >}}).
+
+## Requirements
+
+1. [HiveMQ MQTT CLI](https://hivemq.github.io/mqtt-cli/docs/installation.html) installed on your system.
+
+## Connecting to MQTT Server in MQTT CLI Shell Mode
+
+>Learn how to connect to {{% tts %}} MQTT Server by reading the [MQTT Server]({{< ref "/integrations/mqtt" >}}) guide.
+
+Enter the HiveMQ MQTT CLI shell mode by typing the following command in your terminal:
+
+```bash
+$ mqtt shell
+```
+
+Once in shell mode, you can connect to {{% tts %}} MQTT Server by using the following command:
+
+```bash
+$ con -h hostname -p port -V 3 -u username -pw password
```

Comment:
Add a note that password is an API key, ref `integrations/mqtt/#creating-an-api-key`